tests/common.go

```diff
@@ -2976,12 +3040,48 @@ func CreateS3BackupLocation(name string, uid, cloudCred string, cloudCredUID str
 			},
 		},
 	}
-	//ctx, err := backup.GetPxCentralAdminCtx()
+
 	ctx, err := backup.GetAdminCtxFromSecret()
+	expect(err).NotTo(haveOccurred(),
+	fmt.Sprintf("Failed to fetch px-central-admin ctx: [%v]", err))
```

Comment:
return err here. do not use expect as this wont be publishing log to dashboard and test case wont end

Reply:
done